src/Characters/create.py

```diff
@@ -94,12 +94,11 @@ def create_character(name :str, description: str, type :str, level: int, rarity:
     multipliers["magic"] = scaling[rarity]
 
     #Create character, level them up and give them equipment
-    kind = "?" #what does kind mean/do?
-    character = Character(kind,name,stats,multipliers,description,type.capitalize())
+    character = Character('npc', name,stats,multipliers,description,type.capitalize())
```

Comment:
So this is what kind mean :D (where enum)

Reply:
Yes, and yes, it should probably be enum!